src/lib.rs

```diff
@@ -1503,3 +1516,7 @@ impl<T: Counter, F: Counter> PartialEq<Histogram<F>> for Histogram<T>
 #[path = "tests/tests.rs"]
 #[cfg(test)]
 mod tests;
+
+/// Serialization/deserialization support.
+#[path = "serialization/serialization.rs"]
```

Comment:
Any particular reason you don't name the file `serialization/mod.rs` as is normal for Rust modules, which will let you get rid of this `#[path]` directive?